src/codes.ts

```diff
@@ -94,7 +97,7 @@ class CodesByPurpose extends CodeGroup {
         .map(({ purpose, id }): [Purpose, ChainCode] => [purpose, id]),
     );
     if (codeMap.size !== 2) {
```

Comment:
You're comparing whether it's exactly two, but if it's not, you say there are none. That's rather unexpected. Please change the check compare to 0.

Reply:
Changed the error message